alsa-lib: add support for pistachio-card in alsa-lib.

manohar-narkhede-imgtec opened the pull request:
This fixes CreatorDev/openwrt#25

manohar-narkhede-imgtec committed:
alsa-lib: add support for pistachio-card in alsa-lib.

Signed-off-by: Manohar Narkhede <Manohar.Narkhede@imgtec.com>

abhijit-mahajani-imgtec commented:
CI job is getting failed since JenkinsFile here is missing change like this https://github.com/CreatorDev/Ci40-platform-feed/pull/105

manohar-narkhede-imgtec committed:
Jenkinsfile: use params via their map

Params are also set as top level variables but not on the first
build. Use the params via the map 'params' which is always set
even on the first build.

Signed-off-by: Manohar Narkhede <Manohar.Narkhede@imgtec.com>

Ham22 commented on Jenkinsfile:
```diff
@@ -2,7 +2,7 @@
 properties([
     parameters([
-        string(defaultValue: 'ci40', description: 'OpenWrt branch to test against', \
+        string(defaultValue: 'ci40', description: 'OpenWrt branch to test against',
```
This is not doing what the commit message says because half the commit is missing?

Ham22 replied:
Plus this fix is still in testing in a single feed and shouldn't be added to all feeds yet so just drop the commit. It hasn't fixed anything the builds have started working because its only the first build per branch that fails.

manohar-narkhede-imgtec commented:
Closing the pull request as suggested by Ian.

Ham22 commented:
No he didn't, he said drop the commit that shouldn't be there
